### PR TITLE
Created form section

### PR DESCRIPTION
**Pull request by Progress2002**
Copied index.html and styles.css code from a previous project

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Survey Form</title>
+    <link rel="stylesheet" href="styles.css">
+
+</head>
+<body>
+    <h1 id="title">FreeCodeCamp Survey Form</h1>
+<p id="description">Thank you for taking the time to help us improve the platform </p>
```
Comment (Progress2002) [inline index.html:13]: I think it will be better to add a proper indentation, which should make your code more readable and clean.

**Review by Progress2002 — COMMENTED** [review]
Hi @Progress2002,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

Highlights
✔️ No linter error
✔Good README file

Required Changes ♻️

1. you should add HTML and CSS linter to your project to ensure it checks out your code and detects linter errors if found
2. I noticed no proper commit messages were made. Please ensure to add a proper and descriptive commit message on each step of the project as it makes your project professional and also flows the proper GitHub flow.

  
Check the comments under the review.
Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @Progress2002, 

Good job so far!
You are doing it well 👍🏽 there are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

- Correct use of resume ✔️


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

- [x] Please 🙏🏽 add a **minimum of 2 inline comments** - you will definitely find something that could have been done better!

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+        <input type="checkbox" id="checkbox" name="checkbox" value="checkbox">
+        <label for="checkbox">Wiki</label><br>
+
+        <p>Any comments or suggestions?</p>
+        <textarea name="comment" id="comment" cols="30" rows="10" placeholder="Enter your comment here..."></textarea>
+
+        <button type="submit" id="submit">Submit</button>
+
+</form>
```
Comment (Progress2002) [inline index.html:84]: I think making the form section properly indented in the body would add to the readability of your code.

Comment (Santiago220991) [reply index.html:84]: - [OPTIONAL] Same here kindly add some emojis to make your comments more human-friendly. :space_invader: :space_invader:

File: index.html (hunk 1 above)
Comment (Santiago220991) [reply index.html:13]: - [OPTIONAL] Kindly add some emojis to make your comments more human-friendly. :space_invader: :space_invader:

**Review by Santiago220991 — APPROVED** [review]
Hi @Progress2002 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 




## Highlights
:heavy_check_mark: Inline comments.
:heavy_check_mark: Good summary.

## Optional suggestions

- It is a good idea to include a GIF image in your summary to make it more human-friendly as I did in this summary :smiley: To do this you could install an extension for Chrome called "GIFs for GitHub". :nerd_face: :nerd_face:

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**